Dockerfile

```diff
@@ -2,20 +2,37 @@ FROM ubuntu:trusty
 MAINTAINER Akeem McLennon <akeem@mclennon.com>
 
 USER root
-RUN apt-get -y update
-RUN apt-get -y dist-upgrade
-RUN apt-get -y build-dep phantomjs
-RUN apt-get install -y python flex bison gperf ruby perl \
-  libsqlite3-dev libfontconfig1-dev libicu-dev libfreetype6 libssl-dev \
-  libpng-dev libjpeg-dev
-RUN apt-get -y install git-core
-WORKDIR /usr/local
-RUN git clone https://github.com/AkeemMcLennon/phantomjs.git
+
+RUN mkdir /usr/local/phantomjs
+
 WORKDIR /usr/local/phantomjs
-RUN ./build.sh --confirm
-RUN ln -s /usr/local/phantomjs/bin/phantomjs /usr/bin/phantomjs 
-RUN apt-get autoremove -y
-RUN apt-get clean all
-ADD join-hub.sh /usr/local/phantomjs/
-ENTRYPOINT sh /usr/local/phantomjs/join-hub.sh
 
+ARG CLEAN=0
+
+RUN apt-get -y update \
+      && apt-get -y dist-upgrade \
+      && apt-get -y build-dep phantomjs \
+      && apt-get install -y python flex bison gperf ruby perl \
+                            libsqlite3-dev libfontconfig1-dev \
+                            libicu-dev libfreetype6 libssl-dev \
+                            libpng-dev libjpeg-dev \
+                            git-core libicu52 \
+                            curl \
+      && curl -L https://github.com/AkeemMcLennon/phantomjs/archive/2.0.0.tar.gz | tar --strip-components=1 -xzf - \
```

Comment:
If you're downloading the zip file, you'll need to download from master instead of the 2.0.0 release. Otherwise, the zip file won't contain the ghostdriver patch.